runtime/moonbase/src/asset_config.rs

```diff
@@ -86,6 +85,10 @@ impl pallet_assets::Config<ForeignAssetInstance> for Runtime {
 	type Extra = ();
 	type AssetAccountDeposit = ConstU128<{ currency::deposit(1, 18) }>;
 	type WeightInfo = pallet_assets::weights::SubstrateWeight<Runtime>;
+	type RemoveItemsLimit = ConstU32<1000>;
+	type AssetIdParameter = Compact<AssetId>;
+	type CreateOrigin = AsEnsureOriginWithArg<EnsureNever<AccountId>>;
```

Comment:
Does this effectively prevent us from creating new assets?

Reply:
Oh, that probably makes sense; we can call `force_create()` but not `create()`. Does that sound right?

Reply:
Yeah we're using `force_create` via the `pallet-asset-manager` so it makes sense to disable asset creation from signed origins.